src/functions/newsletter-submission.js

```diff
@@ -10,10 +10,10 @@ exports.handler = async ({ headers, body }) => {
   }
 
   const email = params.get("email");
-  const tag = params.get("tags");
-
-  const tags = [];
-  tags.push(tag);
+  const tags = params
```

Comment:
@will0101 Heads up. I made a quick change here that does two things:
1. Allows us to specify multiple tags in the input: `<input name="tags" value="one,two,three">`
2. Trims any whitespace from them.

Doing the `split` also allows us to do this all at once, instead of creating the `tag` variable and pushing it to an empty array.